servers/pollux/default.nix

```diff
@@ -0,0 +1,163 @@
+{ modulesPath, inputs, config, pkgs, ... }:
+let
+  vs = config.vault-secrets.secrets;
+
+  suitecrm = import ./suitecrm.nix { inherit pkgs; system = pkgs.system; };
+in {
+  imports = [
+    inputs.serokell-nix.nixosModules.ec2
+    inputs.hermetic.nixosModules.hermetic
+  ];
+
+  networking.firewall.allowedTCPPorts = [ 80 443 ];
+
+  users.users.suitecrm = {
+    isNormalUser = true;
```

Comment:
Shouldn't this be `isSystemUser` instead?

Reply:
changed